Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptSearchController.java

```diff
@@ -87,6 +87,8 @@ public String search(HttpServletRequest req, ModelMap model, @RequestParam(defau
         if (results.hasErrors()) {
             return "conceptsearch";
         }
+        String str = conceptSearchBean.getWord().trim();
+        conceptSearchBean.setWord(str);
```

Comment:
The lines 90 and 91 can be merged into a single statement: conceptSearchBean.setWord(conceptSearchBean.getWord().trim())

Reply:
Yes, I'll merge these into single statement

Reply:
lines 90 and 91 can be merged